lib/oli_web/templates/layout/delivery.html.heex

```diff
@@ -160,6 +160,34 @@
 
     <%= Map.get(assigns, :inner_layout) || @inner_content %>
 
+    <div
```

Comment:
I think we already have an existing modal implementation using
`window.oliDispatch(modalActions.display(...`. Granted this one is much simpler, but maybe we should
use the existing one or replace that one to standardize on a single approach so we dont have two
different approaches to displaying modals?